src/main/java/io/codelex/exceptions/practice/Exercise2.java

```diff
@@ -0,0 +1,32 @@
+package io.codelex.exceptions.practice;
+
+public class Exercise2 {
+    public static void main(String[] args) {
+        methodA();
+    }
+
+
+    public static void methodA() throws ArithmeticException {
+        try {
+            methodB();
+        } catch (ArithmeticException e) {
```

Comment:
`After each try{} block put a catch{} block which catches the exception, prints a stack trace, and throws the exception object to its caller`

Reply:
Salabots.